src/index.html

```diff
@@ -1,14 +1,40 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="nav__logo">
+        <img src="./images/logo.png" alt="MOYO shop logo">
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__list-item"><a href="#" class="nav__link is-active">Apple</a></li>
```

Comment:
Please refer to this [guideline](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping) and fix the formatting